tests/dummy/app/components/x-button.js

```diff
@@ -7,6 +7,7 @@ export default Component.extend({
   tagName: 'button',
 
   click() {
+    this.get('raven').captureBreadcrumb('Breadcrumb captured from test component');
```

Comment:
I think Raven.js expects objects for breadcrumbs - see https://docs.sentry.io/clients/javascript/usage/#raven-js-recording-breadcrumbs